crypto/engine/eng_cryptodev.c

```diff
@@ -594,9 +609,13 @@ static const EVP_CIPHER *cryptodev_des_cbc(void)
             || !EVP_CIPHER_meth_set_init(cipher, cryptodev_init_key)
             || !EVP_CIPHER_meth_set_do_cipher(cipher, cryptodev_cipher)
             || !EVP_CIPHER_meth_set_cleanup(cipher, cryptodev_cleanup)
-            || !EVP_CIPHER_meth_set_impl_ctx_size(cipher, sizeof(struct dev_crypto_state))
-            || !EVP_CIPHER_meth_set_set_asn1_params(cipher, EVP_CIPHER_set_asn1_iv)
-            || !EVP_CIPHER_meth_set_get_asn1_params(cipher, EVP_CIPHER_get_asn1_iv)) {
+            || !EVP_CIPHER_meth_set_impl_ctx_size(cipher,
+                                                  sizeof(struct
```

Comment:
as far as I can see, type of `cipher` is not related in any way, `EVP_CIPHER` is type of first argument of `EVP_CIPHER_meth_set_impl_ctx_size` and second parameter is unrelated `int`.